client/src/panels/panel_declarations/finances/goco/goco.js

```diff
@@ -277,9 +331,11 @@ class Goco extends React.Component {
             <NivoResponsiveBar
               { ...nivo_default_props }
               data={ node.data.children }
+              custom_table={_.find(child_tables, ['key',node.indexValue]).table}
               onMouseEnter={ (child_node, e) => handleHover(child_node, e.target, node.data.children) }
               onMouseLeave={ (child_node, e) => handleHover(child_node, e.target, node.data.children) }  
               onClick={ (child_node, e) => window.open(tick_map[child_node.indexValue], '_blank') }
+              graph_height = '500px'
```

Comment:
Seeing `graph_height = '400px'` in most places (is that the default? maybe we shouldn't add it explicitly everywhere). Does this graph have a special reason to get more height?

Reply:
Most yes but not overwhelmingly so. I can make 400px the default but its overridden a fair amount (e.g. for donuts).  Haven't tested it but everything is different for the gocographic so why not this? :P